src/web_interface/templates/database/database_advanced_search.html

```diff
@@ -12,6 +12,19 @@
     </script>
 {% endblock %}
 
+{% block styles %}
+    <style>
+        .example-json {
```

Comment:
Is `example` intendet or a copy-pase artifact?

Reply:
actually it was intentional since it is used for the _advanced search_ examples :sweat_smile: